root/static/scripts/edit/MB/TextList.js

```diff
@@ -18,13 +18,17 @@ MB.Form.TextList = function (input) {
     var $template = $('.' + template.replace(/\./g, '\\.'));
     var counter = 0;
 
-    self.removeEvent = function () {
-        $(this).closest('div.text-list-row').remove();
+
+    self.removeEvent = function (event) {
+        $(this).closest('div.text-list-row')
+            .remove();
+
```

Comment:
This seemed fine how it was, since there were only two method calls in the chain and everything fit in one line.